plugins/gradle/src/main/groovy/com/newrelic/agent/android/ClassTransformWrapperTask.groovy

```diff
@@ -7,6 +7,7 @@ package com.newrelic.agent.android
 
 import com.newrelic.agent.compile.ClassTransformer
 import groovy.io.FileType
+import org.apache.commons.io.IOUtils
```

Comment:
Remove unused imports

Reply:
Yes, it's removed. This is an outdated file.